app/controllers/users_controller.rb

```diff
@@ -1,5 +1,11 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+    render '/users/index'
```

Comment:
- [ ] Please delete this unnecessary line as Rails already have naming convention and the right template will be called when this action fired.

Reply:
lines removed successfully